public/app/features/explore/LiveLogs.tsx

```diff
@@ -82,24 +152,33 @@ class LiveLogs extends PureComponent<Props> {
           <div
             ref={element => {
               this.liveEndDiv = element;
-              if (this.liveEndDiv) {
+              // This is triggered on every update so on every new row. It keeps the view scrolled at the bottom by
+              // default.
+              if (this.liveEndDiv && !isPaused) {
                 this.liveEndDiv.scrollIntoView(false);
               }
             }}
           />
         </div>
         <div className={cx([styles.logsRowsIndicator])}>
-          <span>
-            Last line received: <ElapsedTime resetKey={this.props.logsResult} humanize={true} /> ago
-          </span>
-          <LinkButton
-            onClick={this.props.stopLive}
-            size="md"
-            variant="transparent"
-            style={{ color: theme.colors.orange }}
+          <button
+            onClick={isPaused ? onResume : onPause}
+            style={{ marginRight: theme.spacing.sm }}
```

Comment:
Can we extract this style to a constant in the file? Same custom style is being used on next button.

Reply:
Yes will do that.